run_dir/design/qpcr_pools.html

```diff
@@ -14,7 +14,8 @@
       <h1><span id="page_title">qPCR pools</span>
         <small>Showing all qPCR pools</small>
       </h1>
-      <table class="table table-striped table-bordered sortable" id="pools_table">
+      <label class="btn btn-sm btn-default expand-all"><span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>Expand All</label>
```

Comment:
```suggestion
      <label class="btn btn-sm btn-default expand-all pull-right"><span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>Expand All</label>
```
Also move above the `<h1>` so that it floats on the same line.

![image](https://user-images.githubusercontent.com/465550/74233659-64b6e000-4ccb-11ea-8d45-cfea31e713d9.png)